test/fast/Unit tests/nvm_ls_current

```diff
@@ -19,20 +19,21 @@ fi
 
 rm -rf "$TEST_DIR"
 mkdir "$TEST_DIR"
-ln -s "$(command which which)" "$TEST_DIR/which"
+# Ensure that the system version of which is used, not node_modules/.bin/which
+ln -s "$(PATH="/usr/bin" command which which)" "$TEST_DIR/which"
```

Comment:
i don't think `/usr/bin` will always be where it's located. i think the issue is that `node_modules/.bin` is in the PATH when `npm run` is used - so, can we remove that from the PATH rather than hardcoding one possible location of `which`?